src/swap/index.ts

```diff
@@ -1,10 +1,15 @@
 // 🌲☀️🌲
 export { getQuote } from './core/getQuote';
+export { getSwap } from './core/getSwap';
```

Comment:
@0xAlec more a an open question, is `getSwap` too short as name?

We wrote
```
Retrieves an unsigned transaction for a swap from Token A to Token B.
```

I wonder if `get` is the right verb, and maybe something like `prepareSwapTransaction` or `buildSwapTransaction`.

Let's not change the name yet, but would love to reflect a second on this.

Reply:
I like `buildSwapTransaction`!